java/src/org/openqa/selenium/grid/config/BUILD.bazel

```diff
@@ -22,5 +22,6 @@ java_library(
         artifact("com.beust:jcommander"),
         artifact("com.google.guava:guava"),
         artifact("io.ous:jtoml"),
```

Comment:
Wondering if this should be removed ?

Reply:
Yes, this will be removed when I pass the existing tests.